### ISSUE TITLE
Docker: libm.so.6: version `GLIBC_2.27' not found

### ISSUE THREAD

**Issue by beevelop** [facebook/duckling#535]
```
duckling-example-exe: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by duckling-example-exe)
```

The current (7889f396f324028075d39e5003f1083ad05d68f3) Docker image  seems to have an issue with `glibc`. Referring to https://stackoverflow.com/questions/54616024/error-lib-x86-64-linux-gnu-libm-so-6-version-glibc-2-27 this might be caused by the builder base image using `haskell:8` – which resolves to `8.10.2-buster` as of right now – whereas the "executing" image is based on `debian:stretch-slim`.

TLDR: The build happens with `buster` and the execution fails with `stretch`.

**Comment by chessai** [comment]
this should be resolved in master - feel free to re-open if otherwise.
